frontend/src/marketplace/EntryList.tsx

```diff
@@ -22,49 +23,69 @@ export default function EntryList({
 }: EntryListProps) {
   const theme = useTheme()
 
-  const entriesDisplay = useMemo(() => {
-    return entries.map((entry, index) => (
-      <Fragment key={entry.id}>
-        <Stack direction='row'>
-          <Link
-            sx={{ textDecoration: 'none', whiteSpace: 'nowrap', textOverflow: 'ellipsis', overflow: 'hidden' }}
-            href={`${entry.kind}/${entry.id}`}
-          >
-            <Typography
-              variant='h5'
-              sx={{ fontWeight: '500', textDecoration: 'none', color: theme.palette.primary.main }}
+  const Row = memo(({ data, index, style }: { data: EntrySearchResult[]; index: number; style: CSSProperties }) => {
```

Comment:
If you want to use `memo()`, which is a HOC (Higher Order Component), you should split this out into its own component. 

That being said, I don't really see the performance benefit of memoizing something that's going to be unrendered and rerendered as the user scrolls.

IMO this should be pulled out into a new `EntryListRow` component and we can do away with the `memo`. I do think it's worth checking that there isn't a degradation of performance if we do this, but I wouldn't expect there to be any noticeable change either way. Therefore I'd vote for simplifying by removing the `memo`.